# Context: winston@2.4.7 comes via @hmcts/nodejs-logging and codeceptjs->detox->@wix-pilot/core.
# No newer upstream version is available yet, so we patch the Node 22 util._extend deprecation here.
diff --git a/lib/winston/container.js b/lib/winston/container.js
index 683eee3c5160ef66a82e6102f24a5cb973f29eb2..e48b86fb8f3d59746e39a5c2ec9e5e0cb15ddae1 100644
--- a/lib/winston/container.js
+++ b/lib/winston/container.js
@@ -8,7 +8,8 @@
 
 var common = require('./common'),
     winston = require('../winston'),
-    extend = require('util')._extend;
+    // util._extend is deprecated on Node 22; Object.assign is drop-in for this usage.
+    extend = Object.assign;
 
 //
 // ### function Container (options)
